roles/openshift_master/tasks/systemd_units.yml

```diff
@@ -22,34 +22,12 @@
   changed_when: "'Downloaded newer image' in pull_result.stdout"
   when: openshift.common.is_containerized | bool and not openshift.common.is_master_system_container | bool
 
-# workaround for missing systemd unit files
-- name: Create the systemd unit files
-  template:
-    src: "master_docker/master.docker.service.j2"
-    dest: "{{ containerized_svc_dir }}/{{ openshift.common.service_type }}-master.service"
-  when:
-  - openshift.common.is_containerized | bool and (openshift.master.ha is not defined or not openshift.master.ha) | bool
-  - not openshift.common.is_master_system_container | bool
-  register: create_master_unit_file
-
-- name: Install Master service file
-  copy:
-    dest: "/etc/systemd/system/{{ openshift.common.service_type }}-master.service"
-    src: "{{ openshift.common.service_type }}-master.service"
-  register: create_master_unit_file
-  when:
-  - not openshift.common.is_containerized | bool
-  - (openshift.master.ha is not defined or not openshift.master.ha) | bool
-
-- command: systemctl daemon-reload
```

Comment:
Be aware this file is included in the control plane upgrade as well. See https://github.com/openshift/openshift-ansible/blob/master/playbooks/common/openshift-cluster/upgrades/upgrade_control_plane.yml#L94